src/common/applyLanguage.ts

```diff
@@ -154,6 +154,18 @@ export function applyLanguageToVariants(
   return ret;
 }
 
+export const stringify = (x: any) =>
+  JSON.stringify(
+    x,
+    (key, value) =>
+      key === "source"
+        ? undefined
+        : typeof value === "bigint"
+        ? value.toString() + "n"
+        : value,
+    2
+  );
+
```

Comment:
As a general helper function, it may be best to move stringify to a new file, e.g. `src/common/stringify.ts`